sycl/include/sycl/ext/oneapi/experimental/address_cast.hpp

```diff
@@ -16,6 +16,9 @@ inline namespace _V1 {
 namespace ext {
 namespace oneapi {
 namespace experimental {
+namespace detail {
+using namespace sycl::detail;
+}
```

Comment:
I think it's a bad habit of ours to have any other `detail` namespace than `sycl::detail`. Any details can be in `sycl::detail`, even if it's extension related. That said, not this PR's fault and I have added to those namespaces in the past. Just a bit of venting. 😉